ga4gh/datarepo.py

```diff
@@ -42,6 +45,14 @@ def addReferenceSet(self, referenceSet):
         self._referenceSetNameMap[referenceSet.getLocalId()] = referenceSet
         self._referenceSetIds.append(id_)
 
+    def addOntologies(self, ontologies):
```

Comment:
Why not addOntology(self, ontology)? Again, we should follow the established convention here closely for consistency.